src/Tes.Runner/Events/EventMessage.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation.
```

Comment:
It would be extremely useful if this file were in a project that both `Tes.Runner` and `TesApi.Web` both share, along with an enumeration of the various events currently being const strings in `EventsPublisher.cs`